Adyen/Core/Payment Methods/OnlineBankingPaymentMethod.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// An issuer (typically a bank) in an issuer list payment method.
+public struct Issuer: Decodable, CustomStringConvertible, Equatable {
```

Comment:
Is this the same issuer we have in `IssuerListPaymentMethod`?
Can we reuse this model?

Reply:
yes, we can :).